.codespellrc

```diff
@@ -0,0 +1,7 @@
+[codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = .git*,*.svg,*.css,.codespellrc,locales,pnpm-lock.yaml,i18n,config.ts,extensions.json
+check-hidden = true
+# Case sensitive names etc
+ignore-regex = \b(Manuel|Sie diesen|Sie den|l'authentification)\b
```

Comment:
Whats this for?

Reply:
comment above attempts to describe "Case sensitive names".  I pushed f119d1817d216cf65225be16b43bc7b6ccc694ba to provide more explanation there